api/tests/shared/integration/infrastructure/jobs/JobPgBoss_test.js

```diff
@@ -18,7 +18,7 @@ describe('Integration | Infrastructure | Jobs | JobPgBoss', function () {
     const job = new Job({ name, retryLimit, retryDelay, retryBackoff, expireIn, priority }, knex);
 
     // when
-    await job.schedule(expectedParams);
+    await job.performAsync(expectedParams);
```

Comment:
Tester le résultat également
```suggestion
   const result = await job.performAsync(expectedParams);
```

Reply:
je l'ai mis dans un it à part